pyvrp/cpp/bindings.cpp

```diff
@@ -157,11 +157,25 @@ PYBIND11_MODULE(_pyvrp, m)
                     std::vector<ProblemData::VehicleType> const &vehicleTypes,
                     std::vector<std::vector<pyvrp::Value>> const &dist,
                     std::vector<std::vector<pyvrp::Value>> const &dur) {
-                     Matrix<pyvrp::Distance> distMat(clients.size());
-                     Matrix<pyvrp::Duration> durMat(clients.size());
+                     auto const numNodes = clients.size();
```

Comment:
I want to avoid cluttering the bindings too much, but I don't think there's a better way to check this right now. Maybe once we have support for `py::array` in #61.